pkgs/applications/blockchains/nbxplorer/default.nix

```diff
@@ -17,7 +17,7 @@ buildDotnetModule rec {
   dotnet-sdk = dotnetCorePackages.sdk_3_1;
   dotnet-runtime = dotnetCorePackages.aspnetcore_3_1;
 
-  postInstall = ''
+  preFixup = ''
```

Comment:
Just to make sure, is the usage of `preFixup` instead of `postFixup` here proposital :thinking: ?